library/common/extensions/stat_sinks/metrics_service/service.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package envoymobile.extensions.stat_sinks.metrics_service;
+
+import "envoy/config/core/v3/base.proto";
+import "metrics.proto";
+import "validate/validate.proto";
+
+service EnvoyMobileMetricsService {
+  rpc EnvoyMobileStreamMetrics(stream EnvoyMobileStreamMetricsMessage)
+      returns (stream EnvoyMobileStreamMetricsResponse) {
+  }
+}
+
+message EnvoyMobileStreamMetricsResponse {
+  string batch_id = 1;
+}
+
+message EnvoyMobileStreamMetricsMessage {
+
+  message Identifier {
+    // The node sending metrics over the stream.
+    envoy.config.core.v3.Node node = 1 [(validate.rules).message = {required: true}];
+  }
+
+  // Identifier data effectively is a structured metadata. As a performance optimization this will
+  // only be sent in the first message on the stream.
+  Identifier identifier = 1;
+
+  // A list of metric entries
+  repeated io.prometheus.client.MetricFamily envoy_metrics = 2;
+
+  string batch_id = 3;
```

Comment:
nit: I would make this field 2.

Also lets document what this field is for.

Reply:
you mean 
```
string batch_id = 2;
repeated io.prometheus.client.MetricFamily envoy_metrics = 3;
```
?

Reply:
Yes